tools/nut-scanner/scan_nut.c

```diff
@@ -177,24 +177,22 @@ static void * list_nut_devices(void * arg)
 		/* FIXME:
 		 * - also print answer[2] if != "Unavailable"?
 		 * - for upsmon.conf or ups.conf (using dummy-ups)? */
-		if (numa >= 3) {
```

Comment:
I wonder if this originally intended to check for other constraints on `numa` (e.g. less than some limit?) and behave differently than for values too small as done above...

@aquette @zykh @clepple : any memories on this one?

Reply:
Anyway, this change does simplify code and does not change its practical behavior, so okay :)